src/app/pages/PerpetualPage/hooks/usePerpetual_OpenPositions.ts

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import { PerpetualPairType } from '../../../../utils/dictionaries/perpatual-pair-dictionary';
+import { useBlockSync } from '../../../hooks/useAccount';
+
+export type OpenPositionEntry = {
+  id: string;
+  pair: keyof typeof PerpetualPairType;
+  position: number;
+  value: number;
+  entryPrice: number;
+  markPrice: number;
+  liquidationPrice: number;
+  margin: number;
+  leverage: number;
+  /** [shortAssetValue, longAssetValue, returnOnEquity] */
+  unrealized: [number, number, number];
```

Comment:
I wrote it elsewhere but adjust this, make it an object and remove the comment, it will be more readable.